deploy/basic/Dockerfile

```diff
@@ -121,16 +151,15 @@ FROM base
 ARG oracleClientVersion
 ARG nodeVersion
 WORKDIR /
-# Support for Offline local image. Online image on OCI Object Storage
-COPY src/catalogue/images/ images/
 
 # Copy Services apps
 COPY --from=storefront-builder /app/storefront/build /app/storefront
 COPY --from=api-builder /app/api /app/api
+COPY --from=assets-builder /app/assets /app/assets
 COPY --from=catalogue-builder /catalogue /app/catalogue/catalogue
 
 # Create zip package of the Apps and local images
-RUN mkdir /package && GZIP=-9 tar cvzf /package/mushop-basic.tar.gz /app /images
+RUN mkdir /package && GZIP=-9 tar cvzf /package/mushop-basic.tar.gz /app
```

Comment:
with the new changes, what is the new size of the stack.zip?

Reply:
7.8Mb